my_game.py

```diff
@@ -450,12 +431,22 @@ def on_update(self, delta_time):
                 ufo_hit.destroy()
                 self.player_score += UFO_POINTS_REWARD
 
+        if any(self.player_sprite.collides_with_list(self.asteroid_list)):
```

Comment:
Add a comment on what this block is doing.

Reply:
Have done it says "# Checks if the Player touching any astroid"